test/lib.bash

```diff
@@ -193,17 +193,16 @@ function run_rolling_upgrade_tests {
   patch="${KNATIVE_SERVING_HOME}/openshift/patches/001-object.patch"
   git apply "${patch}"
 
-  image_version=$(versions.major_minor "${KNATIVE_SERVING_VERSION}")
+  # FIXME(ksuszyns): remove static version resolving
+  image_version=0.21
```

Comment:
This is interim change, to be removed when https://github.com/openshift/release/pull/19133 is merged and images are mirrored

Reply:
openshift/release#19133 is merged and images are mirrored

/unhold